fhir-profile/src/main/java/com/ibm/fhir/profile/ConstraintGenerator.java

```diff
@@ -340,6 +340,8 @@ private String generateReferenceTypeConstraint(ElementDefinition elementDefiniti
 
         if (isOptional(elementDefinition)) {
             sb.append(identifier).append(".exists()").append(" implies (");
+        } else {
+            sb.append(identifier).append(".exists()").append(" and ");
```

Comment:
is there a bug this fixes?
worth adding to the summary for the release?

Reply:
No formal issue was created for this but I think it makes sense to add it to the summary.